lib/auth/auth_with_roles.go

```diff
@@ -3462,8 +3462,16 @@ func (a *ServerWithRoles) generateUserCerts(ctx context.Context, req proto.UserC
 		// Role impersonation uses the user's own name as the impersonator value.
 		certReq.impersonator = a.context.User.GetName()
 
-		// Deny reissuing certs to prevent privilege re-escalation.
```

Comment:
More of a nitpick, but it might be useful to tweak the ["identity is not allowed to reissue certificates"](https://github.com/gravitational/teleport/blob/master/lib/auth/auth_with_roles.go#L3201) error message to point users toward this new flag

Reply:
Indeed - do you think it would make more sense to try and intercept this at the client or modify the server error?

Reply:
I think modifying the server error is fine if its easier, no other components use this flag / renewable certs / etc so in practice 100% of the time this will be tbot specific.